app/boards/shields/murphpad/Kconfig.defconfig

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2021 The ZMK Contributors
```

Comment:
I assume all this murphpad stuff was accidentally added?

Reply:
Yes, absolutely. I thought I cleaned it up already.